python/tank/platform/application.py

```diff
@@ -265,36 +265,25 @@ def log_exception(self, msg):
     ##########################################################################################
     # internal API
 
-    def log_metric(self, action, log_version=False, log_once=False):
-        """Logs an app metric.
-
-        :param action: Action string to log, e.g. 'Execute Action'
-        :param log_version: If True, also log a user attribute metric for the
-            version of the app. Default is `False`.
-        :param bool log_once: ``True`` if this metric should be ignored if it
-            has already been logged. Defaults to ``False``.
-
-        Logs a user activity metric as performed within an app. This is a
-        convenience method that auto-populates the module portion of
-        `tank.util.log_user_activity_metric()`.
-
-        If the optional `log_version` flag is set to True, this method will
-        also log a user attribute metric for the current version of the app.
-
-        Internal Use Only - We provide no guarantees that this method
-        will be backwards compatible.
-
+    @property
+    def _metric_properties(self):
+        """
+        :returns: A dictionary with metric properties specific to this application.
```

Comment:
please include an example of what this returns. IMO I find that's super helpful when you read the code.